.gitignore

```diff
@@ -1,4 +1,290 @@
+
+#Meld
+*BACKUP*
+*LOCAL*
+*BASE*
+*REMOTE*
+*.orig
+
+#Sqlite
+*.sqlite3
+
+# Created by https://www.gitignore.io/api/python,django,pycharm+all
+# Edit at https://www.gitignore.io/?templates=python,django,pycharm+all
+
+### Django ###
+*.log
+*.pot
 *.pyc
-/db.sqlite3
-database.sqlite3
+__pycache__/
+local_settings.py
+db.sqlite3
```

Comment:
Aaaaand the database is back? (our file is called `database.sqlite3` not the default `db.sqlite3` because @cytadela8 configured it that way for whatever reason)